e2e-tests/experiments/lvm-localpv-provisioner/lvm_utils_ds.yml

```diff
@@ -22,7 +22,7 @@ spec:
         #test: lvm-vg
       containers:
       - name: lvm-vg-creation
-        image: w3aman/lvmutils:ci
+        image: quay.io/w3aman/lvmutils:ci
```

Comment:
@w3aman  can you push the image into any generic place. instead of pusing to your account?

Reply:
yes, plan is to keep this image in openebs account.
but as of now can we keep it here and later we will migrate it ?